screenpipe-integrations/src/unstructured_ocr.rs

```diff
@@ -14,7 +15,10 @@ use std::path::PathBuf;
 use tempfile::NamedTempFile;
 use tokio::time::{timeout, Duration};
 
-pub async fn perform_ocr_cloud(image: &DynamicImage) -> Result<(String, String, Option<f64>)> {
+pub async fn perform_ocr_cloud(
+    image: &DynamicImage,
+    languages: Vec<Language>,
```

Comment:
to be honest i don't even know if unstructured works, did not try for a while

Reply:
Okay, It uses tesseract. I followed api documentation